keras_cv/datasets/waymo/transformer.py

```diff
@@ -581,55 +581,36 @@ def build_tensors_from_wod_frame(frame: dataset_pb2.Frame) -> Dict[str, tf.Tenso
 
 
 def build_tensors_for_augmentation(
-    input_frames: List[Dict[str, tf.Tensor]]
+    frame: Dict[str, tf.Tensor]
 ) -> Tuple[tf.Tensor, tf.Tensor]:
     """Builds tensors for multi-frame data augmentation from input frames.
 
     Args:
-      input_frames: a list of Waymo Open Dataset frames.
+      input_frames: a dictionary of feature tensors from a Waymo Open Dataset frames.
 
     Returns:
       A tuple of two tensors (pointcloud, boxes), with shape
       ([batch size, num frames, num point, num features],
```

Comment:
The "B" dimension is gone? Also I guess the transformer_test.py should be updated.

Reply:
right -- map functions for the tf data pipeline are easier to use if they operate on individual examples instead of batches, and this allows us to call map directly on the output of the transformer

Reply:
re:tests -- good call. I will update those!